manim/scene/scene.py

```diff
@@ -85,7 +86,7 @@ def __init__(self, **kwargs):
         self.tear_down()
         # We have to reset these settings in case of multiple renders.
         file_writer_config["skip_animations"] = False
```

Comment:
Issue: It should be reset to `self.original_skipping_status`, not to `False`.

Reply:
Hmm fair point, I disagree. scene.original_skipping_status value is set to the one of `file_writer_config[skip_animations]` at the beginning of `__init__`, so it will be set to False in the next scene instantiation.

My concern is when scene.original_skipping_status has been changed for a reason or another. If it is the case and we do your suggestion, then `scene.original_skipping_status` will never be set to False again, for the reason above.